setup.py

```diff
@@ -41,18 +41,11 @@ def find_version(*parts):
             'django-cadmin = configurations.management:execute_from_command_line',
         ],
     },
+    install_requires=['django-environ'],
     extras_require={
-        'cache': ['django-cache-url'],
-        'database': ['dj-database-url'],
-        'email': ['dj-email-url'],
-        'search': ['dj-search-url'],
         'testing': [
             'django-discover-runner',
             'mock',
-            'django-cache-url>=1.0.0',
-            'dj-database-url',
-            'dj-email-url',
-            'dj-search-url',
             'six',
             'Sphinx>=1.4',
         ],
```

Comment:
Could maybe still be in `extras_require`?

Reply:
Looks no need here. How useful to keep those in extra_require?

Reply:
It avoids having to install / installing it by default when not needed.

I guess the typical usecase is using URLs for configs then, so that seems to be OK.